source/application/SXRMB/SXRMBAppController.cpp

```diff
@@ -347,6 +327,11 @@ void SXRMBAppController::setupExporterOptions()
 		AMAppControllerSupport::registerClass<SXRMB2DMapScanConfiguration, AMExporter2DAscii, AMExporterOptionGeneralAscii>(sxrmbExportOptions->id());
 }
 
+void SXRMBAppController::setupUserConfiguration()
```

Comment:
SXRMB has a user configuration.  Not sure why it's not in here.  I noticed a couple of the other beamlines also don't have anything in here.